reference/generators/cmakemulti.rst

```diff
@@ -1,41 +1,131 @@
 .. _cmakemulti_generator:
 
-
-`cmake_multi`
-==============
+cmake_multi
+===========
 
 .. container:: out_reference_box
 
     This is the reference page for ``cmake_multi`` generator.
     Go to :ref:`Integrations/CMake<cmake>` if you want to learn how to integrate your project or recipes with CMake.
 
-Usage
------
-
-.. code-block:: bash
+This generator will create 3 files with the general information and specific Debug/Release ones:
 
-    $ conan install -g cmake_multi -s build_type=Release ...
-    $ conan install -g cmake_multi -s build_type=Debug ...
+- *conanbuildinfo_release.cmake*: Variables adjusted only for build type Release
+- *conanbuildinfo_debug.cmake*: Variables adjusted only for build type Debug
+- *conanbuildinfo_multi.cmake*: Which includes the other two and enables its use and has more generic variables and macros.
 
-These commands will generate 3 files:
+Variables in *conanbuildinfo_release.cmake*
+-------------------------------------------
 
-- ``conanbuildinfo_release.cmake``: Variables adjusted only for build_type Release
-- ``conanbuildinfo_debug.cmake``: Variables adjusted only for build_type Debug
-- ``conanbuildinfo_multi.cmake``: Which includes the other two, and enables its use
+Same as :ref:`conanbuildinfo.cmake<conanbuildinfocmake_variables>` with suffix ``_RELEASE``
 
-Variables in conanbuildinfo_release.cmake
+Variables in *conanbuildinfo_debug.cmake*
 -----------------------------------------
 
-Same as :ref:`conanbuildinfo.cmake<conanbuildinfocmake_variables>` with suffix ``_RELEASE``
+Same as :ref:`conanbuildinfo.cmake<conanbuildinfocmake_variables>` with suffix ``_DEBUG``
 
+Macros available in *conanbuildinfo_multi.cmake*
```

Comment:
I would focus on the differences between `cmake` and `cmake_multi`. This piece of docs is not going to be complete (above you have already referenced to "same as conanbuildinfo.cmake with suffix _DEBUG"), so I think it makes no sense to have here almost the copy/paste.

Maybe we have to document here how this file includes debug/release one, and those _more generic variables and macros_...